src/pages/health-declaration/health-declaration.ts

```diff
@@ -155,7 +151,24 @@ export class HealthDeclarationPage extends BasePageComponent {
         select(getPassCertificateNumber),
       ),
     };
+    this.rehydrateFields();
+  }
+
+  rehydrateFields(): void {
+    this.pageState.healthDeclarationAccepted$
+      .subscribe((val) => {
```

Comment:
Confused as to why we need this? I would have thought we would get this for free or just have to run some change detection instead of subscribing twice?

Also we're not calling `unsubscribe` on any of these subscriptions which may lead to memory leaks.